pkg/apis/mysql/v1alpha1/mysqlcluster_defaults.go

```diff
@@ -68,6 +68,20 @@ func (c *MysqlCluster) setPodSpecDefaults(spec *PodSpec) {
 			},
 		}
 	}
+
+	if len(spec.MetricsExporterResources.Requests) == 0 {
+		spec.MetricsExporterResources.Requests = corev1.ResourceList{
+			corev1.ResourceCPU:    resource.MustParse("10m"),
+			corev1.ResourceMemory: resource.MustParse("32Mi"),
+		}
+	}
+
+	if len(spec.MySQLOperatorSidecarResources.Requests) == 0 {
+		spec.MySQLOperatorSidecarResources.Requests = corev1.ResourceList{
+			corev1.ResourceCPU:    resource.MustParse("10m"),
+			corev1.ResourceMemory: resource.MustParse("64Mi"),
+		}
+	}
```

Comment:
@calind what do you think about setting only requested? What about chosesn default values?

Limit is not set, the user can set those values by themselves.

Reply:
I would be hesitant to let the sidecars consume _unlimited_ amount of resources. As long as users have a way to configure the limit I would enforce a sensible default. This goes into `0.4` release so it's ok to modify the defaults.